yaml/csi-driver/v2.1.1/3par-primera-csp.yaml

```diff
@@ -0,0 +1,185 @@
+# Configuration to deploy the HPE 3PAR Primera Storage CSP service
```

Comment:
we are not yet ready for 2.1.1 files. QA would need validate edge yamls first

Reply:
removed